Sources/Run/Routes/BotRoutes.swift

```diff
@@ -36,6 +36,17 @@ extension Droplet {
         get("test") { req in
             return Response(status: Status(statusCode: 200))
         }
+
+        get("last_subscriber") { req in
+            let subscriber = Subscriber.getLastSubscriber()
+            let dictionary = subscriber!.toDictionary()
```

Comment:
p0: try to never use !. 
basically, if there is a problem it will lead to server crash.
you're much better off doing

guard let dictionary = subscribe.toDictionary() else { 
  return ... 5xx error
}